mozci/push.py

```diff
@@ -982,7 +982,10 @@ def find(runnable_type: str) -> Optional[str]:
 
     @memoize
     def get_regressions(
-        self, runnable_type: str, historical_analysis: bool = True
+        self,
+        runnable_type: str,
+        historical_analysis: bool = True,
+        assume_unclassified_is_intermittent: bool = False,
```

Comment:
You could add a test for this parameter in https://github.com/mozilla/mozci/blob/master/tests/test_regressions.py.

A quick way to do it would be to:
- temporarily set this to True by default;
- run tests/test_regressions.py;
- get the failures;
- add the failing tests as new tests with adjusted expectations.